File: 1786-count-the-number-of-consistent-strings/count-the-number-of-consistent-strings.java
class Solution {
    public int countConsistentStrings(String allowed, String[] words) {
        Set<Character> set = new HashSet<>();
        for(char i:allowed.toCharArray()){
            set.add(i);
        }
        int count = 0;
        for(String k:words){
            int flag = 1;
            for(int i=0;i<k.length();i++){
                if(!set.contains(k.charAt(i))){
                    flag = 0;
                    break;
                }
            }
            count += flag;
        }
        return count;
    }
}

File: 1786-count-the-number-of-consistent-strings/count-the-number-of-consistent-strings.py
class Solution:
    def countConsistentStrings(self, allowed: str, words: List[str]) -> int:
        count = 0
        for word in words:
            isvalid = True
            for i in word:
                if i not in allowed:
                    isvalid = False
                    break
            if isvalid:
                count+=1
        return count